runtime-light/tl/tl-functions.h

```diff
@@ -39,6 +39,8 @@ inline constexpr uint32_t DIGEST_SIGN_MAGIC = 0xd345'f658;
 inline constexpr uint32_t DIGEST_VERIFY_MAGIC = 0x5760'bd0e;
 inline constexpr uint32_t CBC_DECRYPT_MAGIC = 0x7f2e'e1e4;
 inline constexpr uint32_t CBC_ENCRYPT_MAGIC = 0x6d4e'e36a;
```

Comment:
It looks like these magics are not needed outside of `store/fetch` implementations. Can we hide them?

Reply:
Yes, but I don't think it's a point, for now at least. Confdata magics should be hidden in this case, too, but there is out of scope of current PR